code/src/java/plugin/pretokens/test/PreEquipTester.java

```diff
@@ -70,7 +71,7 @@ public int passes(final Prerequisite prereq, final PlayerCharacter character, CD
 				}
 				else if (targetEquip.startsWith("TYPE=") || targetEquip.startsWith("TYPE.")) //$NON-NLS-1$ //$NON-NLS-2$
 				{
-					StringTokenizer tok = new StringTokenizer(targetEquip.substring(5).toUpperCase(), ".");
+					StringTokenizer tok = new StringTokenizer(targetEquip.substring(5).toUpperCase(Locale.ENGLISH), ".");
```

Comment:
This is also data dependent, as may be items lower down in this file and later in this PR.